src/js/components/VoterGuide/OrganizationCard.jsx

```diff
@@ -137,16 +137,45 @@ export default class OrganizationCard extends Component {
       if (this.state.organizationPosition.vote_smart_rating) {
         positionDescription = (
           <PositionRatingSnippet
-            {...this.state.organizationPosition}
+            ballotItemDisplayName={this.state.organizationPosition.ballot_item_display_name}
```

Comment:
For each of these components (PositionRatingSnippet, PositionSupportOpposeSnippet, PositionInformationOnlySnippet), make sure the explicit incoming variables are listed in `propTypes` within each component, and the updated camelCase variables replace the prior snake_case format throughout the entire components.

Reply:
Since each of these components is included multiple places in the code (`<PositionInformationOnlySnippet` is shown in 4 places), you will need to resolve the prop type spreading in all of those locations too. Resolving prop type spreading is like a recursive problem.